lightning/src/ln/peer_handler.rs

```diff
@@ -632,7 +648,11 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, OM: Deref, L: Deref, CM
 	/// Constructs a new PeerManager with the given message handlers and node_id secret key
 	/// ephemeral_random_data is used to derive per-connection ephemeral keys and must be
 	/// cryptographically secure random bytes.
-	pub fn new(message_handler: MessageHandler<CM, RM, OM>, our_node_secret: SecretKey, ephemeral_random_data: &[u8; 32], logger: L, custom_message_handler: CMH) -> Self {
+	///
+	/// `current_time` should be set higher than any previous `PeerManager` instance with the same
+	/// secret key. The simplest way to ensure this, of course, is to simply set it to the current
+	/// UNIX timestamp on startup.
```

Comment:
This isn't quite accurate or at least is unclear what is meant by "any previous `PeerManager` instance". It needs to be set higher than what was used to previously initialize plus the number of times that `broadcast_node_announcement` had previously been called. Maybe just say it should set to the current timestamp?